api-extractor.json

```diff
@@ -45,7 +45,7 @@
    *
    * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
    */
-  "mainEntryPointFilePath": "<projectFolder>/lib/esm/index.d.ts",
+   "mainEntryPointFilePath": "<projectFolder>/lib/esm/src/index.d.ts",
```

Comment:
For some reason something related to this change caused build errors where it could not find the path. /lib/esm/index.d.ts. After searching for this file, I found it at "/lib/esm/src/index.d.ts

Reply:
Hmm, I see. I am not sure about what the right thing to do here is. I really don't think any config files should be changed. Let's loop back to this on the index PR since I have a feeling we can't really investigate why until all the pieces are together.